profiles/source/Source_FPD.yaml

```diff
@@ -13,10 +14,10 @@ dsFPD:
       DEFAULT_COLOR_MODE : 0          # Color Mode of LED (Single(0) or Multicolored(1))
       MULTI_COLOR_MODE : 2
```

Comment:
dont need multi color mode , decision can be made based on the default color mode .

Reply:
removed